test/test_markup.rb

```diff
@@ -741,6 +741,17 @@
     output = Gollum::Markup.new(page).render
     assert_html_equal %{<p>a <a href="/alpha.jpg">Alpha</a> b</p>}, output
   end
+  
+  test "page link with relative path" do
```

Comment:
Shouldn't we also test the other case, namely where `LinkedRelative.md` lives in the root?